CONTRIBUTING.rst

```diff
@@ -30,21 +29,24 @@ These drivers are supported on a best effort basis by the developers of the indi
 Note that, any pull request to the main QCoDeS repository concerning unsupported
 drivers will not be reviewed and/or merged with the QCoDeS core.
 
-Bugs reports and feature requests
----------------------------------
+Bugs reports
+------------
 
 We use github's `issues <https://github.com/QCoDeS/Qcodes/issues>`__.
-Search for existing and closed issues. If your problem is not yet
-addressed, `please open a new issue
+If your problem is not yet addressed in the current issues, `please open a new issue
 <https://github.com/QCoDeS/Qcodes/issues/new>`__.
 
-The github GUI will show you a template for bugs.
-Delete the wrong part and try to follow the template. Writing a good
-issue helps you in the first place. Bug reports must be accompanied by a
-reproducible example.
+The github GUI will show you a template for bug reports.
+Please fill in the relevant sections of the template and delete the
+sections  that do not apply to your bug. please include a reproducible
+example of this bug with your report so we can investigate it.
+By writing a good report, we are better able to help you and everyone
+in the QCoDeS community.
```

Comment:
```suggestion
sections that do not apply to your bug. Please include a reproducible
example of this bug with your report so that we can investigate it.
By writing a good report, we are better able to help you and everyone
in the QCoDeS community.
```